about-the-apprentices/2022/Marissa.md

```diff
@@ -0,0 +1,13 @@
+# Hello there
+
+Hello everyone! My name is Marissa and I'm a new apprentice at Sparkbox! Right now, I'm working remotely from Troy, MI, which means I'm on Eastern time and my preferred pronouns are she/her/hers.
+
+One of my favorite things about programming is the building and creating aspect. One thing I'm excited about learning is React...and Git...and anything backend related since I don't have any experience with that...there's a lot I'm excited about!.
```

Comment:
You've got an extra period at the end of this line!

Reply:
good catch! should be fixed i hope!